.github/workflows/ci.yml

```diff
@@ -97,9 +81,48 @@ jobs:
           compression-level: 0  # wheels are ZIP archives
           retention-days: 7
 
+  integration_tests:
```

Comment:
@odinuv this part is vibe-coded so pls look at it - the goal is to not run integration tests in parallel for various python versions since they all work with the same project.

Reply:
Suggestion from @odinuv on Slack:

> in this case (since there is a limited number of those and they are added consciously) i'd say - make multiple projects and add the storage token as part of the test matrix variable